internal/lsp/completion.go

```diff
@@ -70,7 +70,12 @@ func toCompletionItem(candidate lang.Candidate, caps lsp.CompletionClientCapabil
 		Documentation:       doc,
 		TextEdit:            textEdit(candidate.TextEdit, snippetSupport),
 		Command:             cmd,
-		AdditionalTextEdits: textEdits(candidate.AdditionalTextEdits, snippetSupport),
+		AdditionalTextEdits: TextEdits(candidate.AdditionalTextEdits, snippetSupport),
+		// TODO set deprecated via `tags` if supported.
```

Comment:
Thanks for adding the TODO, is it worth moving it below, where we use the legacy `DeprecatedSupport` bool field?

Reply:
I missed below. We're already using tags